custom_components/petlibro/button.py

```diff
@@ -163,6 +163,18 @@ class PetLibroButtonEntityDescription(ButtonEntityDescription, PetLibroEntityDes
 
     ],
     PolarWetFoodFeeder: [
+        PetLibroButtonEntityDescription[PolarWetFoodFeeder](
```

Comment:
When introducing a new name, aka like "Ring bell", make sure to also include it in `translations/en.json`.